From c6e47606cb485457afd06d74c7f372011184fa73 Mon Sep 17 00:00:00 2001
From: Ilya Tumaykin <itumaykin@gmail.com>
Date: Tue, 23 May 2017 17:24:29 +0300
Subject: [PATCH 05/20] Add missing include in src/db.c

url_fclose() function used in this file is defined in fopen.h.
---
 src/db.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/db.c b/src/db.c
index dd133d4..858240d 100644
--- a/src/db.c
+++ b/src/db.c
@@ -28,6 +28,10 @@
 #include "db_disk.h"
 #include "md.h"
 
+#ifdef WITH_CURL
+#include "fopen.h"
+#endif
+
 #ifdef WITH_PSQL
 #include "db_sql.h"
 #endif
-- 
2.19.1

